gui/src/pages/NewProcess.svelte

```diff
@@ -49,6 +50,22 @@
     });
   };
 
+  let mode = 'script';
+  const editorModes = [
+    {
+      id: 'script',
+      name: 'Script',
+      effect: updateScript,
```

Comment:
let's rename this to something clearer with an `on` prefix. For example: `onActivate`